code_schemes/s07e02.json

```diff
@@ -0,0 +1,152 @@
+{
+  "SchemeID": "Scheme-220e6a5e",
+  "Name": "S07E02",
+  "Version": "0.0.0.1",
+  "Codes": [
+
```

Comment:
```suggestion
```

Reply:
I left this blank line deliberately, to hint that that the scheme was incomplete and the normal codes would go here. 

I wish json supported comments...